data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/server/PeerForwarderHttpService.java

```diff
@@ -35,25 +37,29 @@
  */
 public class PeerForwarderHttpService {
     private static final Logger LOG = LoggerFactory.getLogger(PeerForwarderHttpService.class);
+    static final String REQUEST_PROCESSING_LATENCY = "requestProcessingLatency";
```

Comment:
We should probably use the term `server` in this name somewhere. It not clear at immediate reading how this is different from `requestForwardingLatency`.

Maybe `serverRequestTime` vs. `clientForwardingRequestTime`?